src/plugins/appeals/unbanThread.ts

```diff
@@ -0,0 +1,35 @@
+import {useEvent} from "../../hooks";
+import {Interaction, Message} from "discord.js";
+
+useEvent("interactionCreate", async (interaction: Interaction) => {
+    if (!interaction.isButton()) {
+        return;
+    }
+    const {customId} = interaction;
+    const [interactionType, action, userTag, userId] = customId.split("-");
+    const {message} = interaction;
+    if (interactionType !== "appeal") {
+        return;
+    }
+    if (action !== "thread") {
+        return;
+    }
+    try {
+        await interaction.message.startThread ({
+            name: `Appeal Thread - ${userTag}`
+        })
+    } catch {
+        await interaction.reply({
+            content: `A thread was already created for <@${userId}>.`,
+            ephemeral: true,
+        });
+        return;
+    }
+    if (message instanceof Message) {
```

Comment:
There are no cases where `message` is not of type `Message` - this statement will always evaluate to true

Reply:
Moved the silent reply in this case below the message.startThread and wrapped it in a try block.